src/compiler/types.ts

```diff
@@ -3183,6 +3183,8 @@ namespace ts {
         InTypeAlias                             = 1 << 23,    // Writing type in type alias declaration
         InInitialEntityName                     = 1 << 24,    // Set when writing the LHS of an entity name or entity name expression
         InReverseMappedType                     = 1 << 25,
+
+        /* @internal */ DoNotIncludeSymbolChain                 = 1 << 25,    // Skip looking up and printing an accessible symbol chain

```

Comment:
This has the same value as the one above it?

Reply:
:facepalm: yes, yes it does.